tests/PBurggraf/LicensePlate/GermanyDetectorTest.php

```diff
@@ -348,6 +351,8 @@ public function testType($plate, $validity, $type)
             $result = $typeResult->getType();
             $this->assertEquals($type, $result, sprintf('Tested plate: \'%s\', should be %s', $plate, $type));
         }
+
+        $this->assertGreaterThanOrEqual(0, count($typeResults));
```

Comment:
On some test cases, `$typeResults` will be empty array and it will present `This test did not perform any assertions` during PHPUnit tests running.

To fix this issue, it should add this line to let all results can assert length is greater than or equal to `0`.